e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -120,35 +129,74 @@
               "inputName": {
                 "value": "input2"
               },
+              "testResultCoordinatorUrl": {
+                "value": "http://testResultCoordinator:5001/api/testoperationresult"
+              }
+            },
+            "settings": {
               "image": "<Container_Registry>/microsoft/azureiotedge-relayer:<Build.BuildNumber>-linux-<Architecture>",
               "createOptions": ""
             },
-            "networkController": {
+            "testResultCoordinator": {
               "version": "1.0",
               "type": "docker",
               "status": "running",
               "restartPolicy": "always",
+              "env": {
+                "trackingId": {
```

Comment:
Do you think we should add "testStartDelay" here; so that testresultcoordinator should start generating report after (TestStartDelay + TestDuration + fixed buffer to allow all messages received in IoT Hub) or just use (TestDuration + fixed buffer) in code

Reply:
It has a config StartAfter which is default 5 mins so it does not go offline earlier

---
In reply to: [357395326](https://github.com/Azure/iotedge/pull/2112#discussion_r357395326) [](ancestors = 357395326)